scripts/halium

```diff
@@ -1,5 +1,11 @@
 # Local filesystem mounting			-*- shell-script -*-
 
+_log_msg()
```

Comment:
Where is this function being executed?

Reply:
This overwrites the basic log functions from script/functions which is used by log_begin, log_end, etc:
https://anonscm.debian.org/cgit/kernel/initramfs-tools.git/tree/scripts/functions

Reply:
Ah ok, sorry